endgame/alphabeta/alphabeta.go

```diff
@@ -413,6 +441,10 @@ func (s *Solver) Solve(plies int) (float32, []*move.Move, error) {
 				log.Debug().Msgf("Spread at beginning of endgame: %v", s.game.CurrentSpread())
 				log.Debug().Msgf("Maximizing player is: %v", s.game.PlayerOnTurn())
 				s.currentIDDepth = p
+				if p == plies {
+					// Iterative deepening is done
+					s.iterativeDeepeningOn = false
```

Comment:
don't know if we should turn this off; do we want to turn it back on every time we do a new endgame?

Reply:
Ah, that's a good point... I'll merge the `int depth, int plies` change from my other branch which obviates this.